packages/hosted-credit-card-integration/src/components/HostedCreditCardComponent.test.tsx

```diff
@@ -0,0 +1,355 @@
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
```

Comment:
We avoid using `render` from `@testing-library/react` directly.
Please use `renderWithoutWrapper` from `@bigcommerce/checkout/test-utils` for inheirting settings like `legacyRoot`.